docs/pulse/introduction.md

```diff
@@ -81,47 +81,76 @@ We have some suggestions on how to interpret Pulse in a scientifically-sound way
+| metric_dimension | The subcategory of the metric.  For example, if you log value in LogEvent, then value will show up as a subdimension.  dimension = !statsig_topline indicates that this row reflects an aggregate across all dimensions.
+| start_date | The start date for this measurement |
+| end_date | The end date for this measurement |
+| test_units | The number of users in the test group |
+| test_mean | The average value of this metric across test users (or participating units when applicable) |
+| test_stderr | The standard error for the estiamte of the mean for test users.  This can be used to compute confidence intervals. |
+| ctrl_units | The number of users in the control group |
+| ctrl_mean | The average value of this metric across control users (or participating units when applicable) |
+| ctrl_stderr | The standard error for the estiamte of the mean for control users.  This can be used to compute confidence intervals. |
+| abs_delta | The absolute difference between the test and control mean (test_mean - ctrl_mean) |
+| abs_stderr | The estimated standard error of abs_delta |
+| rel_delta | The relative difference between test and control mean, sometimes referred to as lift (test_mean - ctrl_mean)/ctrl_mean |
+| rel_stderr | The estimated standard error of rel_delta  (abs_delta/ctrl_mean) |
+| z_score | The calculated Z-score |
+   
+### Pulse Summary File Description - For Experiments
+   
+| Column Name | Description |
+|-------------|-------------|
+| name | Name of the Experiment or Feature Gate |
+| rule | Name of the Feature Gate Rule. |
+| experiment_group | The group of users for which this metric is computed for.  For a feature gate, this is pass/fail.  For an experiment, this is the variant name. |
+| metric_type | Category of the metric.  Different metric_types are computed differently, including how they're computed in Pulse. |
+| metric_name | The name of the metric.  For event metrics, this is the name of the event. |
+| metric_dimension | The subcategory of the metric.  For example, if you log value in LogEvent, then value will show up as a subdimension.  dimension = !statsig_topline indicates that this row reflects an aggregate across all dimensions. |
+| start_date | The start date for this measurement |
+| end_date | The end date for this measurement |
+| units | The number of users included in this metric estimate. |
+| mean | The average value of this metric across units (or participating units when applicable) |
+| stderr | The standard error for the estiamte of the mean.  This can be used to compute confidence intervals. |
 
 ### First Exposures File Description
 
 | Column Name | Description |
 |-------------|-------------|
-| user_id / stable_id | Refers to the unit identifier used in the experiment |
+| user_id / stable_id / <custom_id> | Refers to the unit identifier used in the experiment |
```

Comment:
why is custom_id in <>?  Because it could be one of many custom IDs?

Reply:
The column name will be user_id or stable_id depending on the configuration.  But if someone has a custom_id called "VIN", we will use that for the column name.